### ISSUE TITLE
Remove extraneous h3 from Collection context tab

### ISSUE THREAD

**Issue by jvine** [projectblacklight/arclight#926]
The "Collection context" tab has a redundant `h3` heading. There is a SR-only `h2` that reiterates the tab label; the `h3` is unnecessary and should be removed.

<img width="384" alt="Screen Shot 2019-10-08 at 10 22 38 AM" src="https://user-images.githubusercontent.com/6945691/66418033-16f53880-e9b6-11e9-8df9-10206a66375e.png">
